src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -134,7 +134,9 @@ private function addTwigOptions(ArrayNodeDefinition $rootNode): void
                     ->example('Twig\Template')
                     ->cannotBeEmpty()
                 ->end()
-                ->scalarNode('cache')->defaultValue('%kernel.cache_dir%/twig')->end()
+                ->scalarNode('cache')
+                    ->setDeprecated('symfony/twig-bundle', '7.3', 'Setting the "%path%.%node%" configuration option is deprecated. It will be removed in version 8.0.')
+                ->end()
```

Comment:
Keeping this option is not a problem, now that all values are correctly supported (true, false, path).

```suggestion
                ->scalarNode('cache')->defaultTrue()->end()
```

Reply:
What would be the use case to change the cache path ?

Reply:
To me, correctly supporting all the values is important to not introduce BC. But if the complexity can be reduced by removing the option entirely, it's better.

Reply:
I don't think there's much complexity in allowing cache deactivation or configuration. The complexity is already imposed by the need to have a different cache in dev and prod.

For usage purposes, you can disable the cache to use your own implementation, for example. Or choose a specific path other than build dir / cache dir.

Reply:
Alright, I still disagree with that, but there's no reason to block the whole PR for that :)

I removed the deprecation, is there anything else missing ?

Reply:
Thanks, we can reconsider depreciating this option in a subsequent PR.